my-app/src/components/TasksList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { deleteTaskAction} from '../redux'
+
+
+const TasksList = () => {
+    const tasks = useSelector((state) => state.tasks)
+    const dispatch = useDispatch()
+    const deleteTask = (id) => dispatch(deleteTaskAction(id))   
+    
+    const tasksList = (tasks.length) ? (
+        tasks.map(task => {
+            return (
+                <div className="collection-item" key={task.id}>
+                <span>{task.content}</span>
+                <button onClick={() => deleteTask(task.id)}>x</button>
```

Comment:
and then you can use `onClick={deleteTask(task.id)}` without redefining this function